src/polyswarmd/__init__.py

```diff
@@ -133,7 +133,7 @@ def before_request():
 
     # Ignore prefix if present
     try:
-        api_key = request.headers.get('Authorization').split()[-1]
+        api_key = request.headers.get('Authorization', '').split()[-1]
```

Comment:
This seems fine but I think we should prefer a broad exception handler that sends them down the unauth path on any error

Reply:
Heh, alright, so just revert that "fix" then I guess.

Reply:
Sure, can do that. Pushed.